SPECS/grub2/remove-shim-lock-verification.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/grub-core/kern/efi/sb.c b/grub-core/kern/efi/sb.c
```

Comment:
nit - consider using a git patch to generate these. It will also capture the commit log message where you can detail the reason for this change.